classes/Hook.php

```diff
@@ -89,19 +91,46 @@ public function add($autodate = true, $null_values = false)
         return parent::add($autodate, $null_values);
     }
 
+    public static function isDisplayHookName($hook_name)
+    {
+        $hook_name = strtolower($hook_name);
+
+        if ($hook_name === 'header') {
+            // this hook is to add resources to the <head> section of the page
+            // so it doesn't display anything by itself
+            return false;
```

Comment:
Actually, this is a display hook. Modules used to **return** a string with the `<script>` & `<link>` elements they want to add to the `<head>` section